modules/dailymotionBidAdapter.js

```diff
@@ -0,0 +1,57 @@
+import { config } from '../src/config.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { VIDEO } from '../src/mediaTypes.js';
+
+export const spec = {
+  code: 'dailymotion',
+  gvlid: 573,
+  supportedMediaTypes: [VIDEO],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   * The only mandatory parameters for a bid to be valid are the api_key and position configuration entries.
+   * Other parameters are optional.
+   *
+   * @param {object} bid The bid to validate.
```

Comment:
This param seems not longer exist

Reply:
Yes, I removed it from the code as we don't use it but maybe we can keep the doc if we need it for future usage?

Reply:
If it's not used I think we shouldn't keep it in documentation to not add ambiguity or fake information when someone need to use this method.

Or if we would keep it (because it was used in the past), we could put again this param in the function and just make a `console.warn` with a message to prevent of the deprecation of this param ? As you want